reference/recognition.md

```diff
@@ -0,0 +1,28 @@
+## Recognition
+
+```
+Recognition {
+  SFFloat  maxRange       100     # maximum distance at which objects are recognized (m)
+  SFInt32  maxObjects     -1      # maximum number of objects detected simultaneously
+  SFBool   occlusion      TRUE    # should occlusions between the camera and the objects be checked
+  SFColor  frameColor     1 0 0   # color of the objects frame in the overlay
+  SFInt32  frameThickness 1       # thickness of the objects frame in the overlay (pixels)
+}
+```
+
+### Description
+
+The [Recognition](#recognition) node provides a [Camera](camera.md) device with object recognition capability.
+When a [Camera](camera.md) device has a [Recognition](#recognition) node in its `recognition` field, it is able to recognize which objects are present in the camera image. Only [Solids](solid.md) whose `recognitionColors` field is not empty can be recognized by the camera.
+
+### Field Summary
+
+- The `maxRange` field defines the maximum distance at which an object can be recognized. Objects farther than `maxRange` are not recognized.
+
+- The `maxObjects` field defines the maximum number of objects detected by the camera. `-1` means no limit. If more objects are visible by the camera, only the `maxObjects` biggest ones (considering pixel size) are recognized.
+
+- The `occlusion` field defines if occlusions between the camera and the object should be checked. Disabling the occlusions detection can save computational time and therefore speed up the simulation, but it can lead to recognized object that are not really visible to the camera.
```

Comment:
As you told me the impact on the simulation speed is low. What is interesting in disabling occlusion is that recognition can work through thin or transparent objects. We should probably explain this.